.circleci/config.yml

```diff
@@ -0,0 +1,33 @@
+version: 2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:12
+    steps:
+      - checkout
+      - run:
+          name: Create Checksum
+          command: sh .circleci/checksum.sh /tmp/checksums.txt
+      - restore_cache:
+          keys:
+            - npm-cache-{{ checksum "/tmp/checksums.txt" }}
```

Comment:
This should probably be done by package instead, which will be a lot faster since only the modules for that package will be redownloaded.

Reply:
Ignore this comment, just saw in the PR it's simple on purpose.